browser/net/brave_ad_block_tp_network_delegate_helper_unittest.cc

```diff
@@ -43,3 +130,21 @@ TEST(BraveAdBlockTPNetworkDelegateHelperTest, DevToolURL) {
   EXPECT_TRUE(request_info->new_url_spec.empty());
   EXPECT_EQ(rc, net::OK);
 }
+
+TEST_F(BraveAdBlockTPNetworkDelegateHelperTest, SimpleBlocking) {
```

Comment:
I added a test fixture which allows the `friend class` to work, and allows common setup/teardown of `g_brave_browser_process` (among other components).